src/components/pages/settings.yml

```diff
@@ -22,3 +22,6 @@ createDocumentRegenerationBatchHeader:
 createDocumentRegenerationBatchButton:
   en: Regenerate documents
   de: Dokumente neu erstellen
+blueprintDocumentHeader:
+  en: Blueprint document
+  de: Blueprint document
```

Comment:
Maybe "Template for new documents" / "Vorlage für neue Dokumente"

Reply:
Agreed, will rename `documentBlueprint` to `documentTemplate` in code as well.
Should we also rename the `blueprintKey` used so far in the cloning logic? I would like to do that.